SharedUtils/src/main/java/com/braintreepayments/api/NetworkResponseCallback.kt

```diff
@@ -0,0 +1,13 @@
+package com.braintreepayments.api
+
+import androidx.annotation.MainThread
+import androidx.annotation.RestrictTo
+
+/**
+ * @suppress
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface NetworkResponseCallback {
```

Comment:
Any chance we could rename this to `HttpResponseCallback`? The `Http` prefix is helpful to call out the grouping of other HTTP related files.